models/r1_rover_mecanum/model.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<model>
+  <name>r1_rover</name>
+  <version>1.0</version>
+  <sdf version='1.5'>r1_rover.sdf</sdf>
+  <license>https://github.com/aionrobotics/aion_io/blob/master/LICENSE.txt</license>
```

Comment:
This path doesn't seem to be valid. Can you recheck if this license is still applicable?

Reply:
Seems like the aion robotics repo is now private.
Maybe @PerFrivik knows more about the license ?